src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala

```diff
@@ -220,6 +227,27 @@ object MCSUtils {
   }
 
   /**
+   * Write nodes to hdfs
+   * @param hdfsDir  String The hdfs directory
+   * @param localDir The local directory
+   * @param filename The filename to be copied
+   */
+  def copyNodesToHDFS(hdfsDir: String, localDir: String, filename: String): Unit = {
```

Comment:
Hrm.. on that note this is indeed generic functionality. It should be in https://github.com/SciSpark/SciSpark/blob/master/src/main/scala/org/dia/utils/FileUtils.scala. Good catch @rahulpalamuttam! 
I will rename to copyFileToHDFS add the copyFileFromHDFS function in the update to this PR. That functionality is useful for viz purposes.  
The function checkHDFSWrite in src/main/scala/org/dia/algorithms/mcs/MCSUtils.scala might also be better placed in FileUtils.